### PR TITLE
DATA-1091 - Docs update: Public data API documentation

### PR DESCRIPTION
**Pull request by tahiyasalam**
Just a quick first go, very open to suggestions!

### CODE REVIEW

File: proto/viam/app/data/v1/data.proto [hunk 1]
```diff
@@ -374,18 +382,22 @@ message RemoveBoundingBoxFromImageByIDRequest {
...
 message BoundingBoxLabelsByFilterRequest {
   Filter filter = 1;
 }
 
+// BoundingBoxLabelsByFilterRequest returns all the labels of the bounding boxes from files from a given filter
 message BoundingBoxLabelsByFilterResponse {
   repeated string labels = 1;
 }
 
+// GetDatabaseConnectionRequest returns the database connection hostname endpoint for a given organization
```
Comment (tahiyasalam) [inline proto/viam/app/data/v1/data.proto:395]: @maxhorowitz Could you look here and below to make sure the docs make sense?

**Comment by tahiyasalam** [comment]
bumping this @agreenb & @maxhorowitz :-)

File: proto/viam/app/data/v1/data.proto (hunk 1 above)
Comment (agreenb) [reply proto/viam/app/data/v1/data.proto:411]: [nit] 'requests the database connection hostname'. The RPC already specifies that it's for ADF so agree that don't need to repeat here.

**Review by agreenb — APPROVED** [review]
This all looks great, thank you so much for the effort in adding this documentation! +💯 for improved DX 🚀

**Pull request merged**